### ISSUE TITLE
Jména speakerů

### ISSUE THREAD

**Issue by PetrCejpek** [miracleflame/Leisure-Suit-Larry-6-CZ-translation-#8]
Některá jména speakerů jsou nepřeložená - hlavně ty holky: Gammie Boysulay -> Gammie Božoléová, Burgundy -> Burgunda, atd. Hledal jsem to v přiložených *.hep souborech, ale tyhle jsem tam neviděl.

**Comment by miracleflame** [comment]
Našiel som si po sebe mail, kde píšem, že sa mi podarilo preložiť mená talkerov vyexportovaním tzv. heap súborov a v nich hexeditom upraviť mená. Skúsil som teraz vyexportovať všetky hep súbory, je ich spústa. Neviem podľa čoho som dokázal usúdiť, v ktorých je meno talkera a v ktorých nie je. Možno podľa reťazca "tBust", ten majú všetky hep v preklade. Každopádne nenašiel som reťazec "Boysulay" v žiadnom. "Burgundy" som našiel v 1804.hep

Asi bude treba menovite hľadať cez Find in files funkciu Notepad++

**Comment by PetrCejpek** [comment]
Zkusil jsem si vyexportovat a editovat ty *.hep soubory, bohužel bez kladného výsledku.

Vysledoval jsem, že ty speakeři v původních *.hep souborech (ty soubory, které byly doposud v rezpozitáři) jsou dialogy, které se vlastně nespustí kurzorem rozhovoru. To znamená například:
**Hlas ženy/Hlas muže** - dialog, který se ozve, když Larry zaťuká někomu na dveře pokoje.
**Domovník Carlos** - ten se ozve, když se mu zavolá přes telefon
**Shalo, Biff Stiff** - to jsou lidi z intra

Zkoušel jsem se podívat na skripty ve SCI Companionu a našel jsem skripty s názvy postav (1800-1816.scr, vlastně to odpovídá těm *.hep souborům). Zkusil jsem všechny skripty dekompilovat a najít stringy s názvy postav. Vyskytují se jen ve výše zmíněných skriptech. Nemůže to být tady?
Nicméně zkoušel jsem to otestovat u Rose (->Rosé), ale bez výsledku. Ovšem nezaručuju, že jsem to udělal dobře. :-D

S tím "Boysulay" se omlouvám, to se tam ani zobrazovat nemá, proto jsi to nikde nenašel.

Co jsou vlastně zač soubory *.hep? Nějaké proměnné, které se cpou do skriptů?

**Comment by PetrCejpek** [comment]
Aha, tak beru zpět část svého předchozího příspěvku. Nakonec se ty *.hep soubory propsaly do hry. (asi jsem to neměl předtím uložené). Ještě to ale ověřím na vícero místech.

**Comment by PetrCejpek** [comment]
![image](https://github.com/user-attachments/assets/6a4869a7-fd0d-4028-beca-d06ef6320f39)
![image](https://github.com/user-attachments/assets/65f44689-c565-4aaa-8372-a2e35ff3d9c8)
![image](https://github.com/user-attachments/assets/2a4c626e-c15a-49dd-97af-661053a5caad)
![image](https://github.com/user-attachments/assets/d4de0e62-898b-4aa9-8361-8aff7282d5c0)

Tak to funguje (předpokládám, že Burgunda bude taky fungovat). Předchozí problém s nepropsáním byl nejspíš způsobený tím, že jsem sice měl změny v souboru uložené, ale pozici ve hře jsem měl uloženou přímo v té "rozhovorové" obrazovce.

**Comment by miracleflame** [comment]
Áno, heap exporty chápem ako nejaký určitý druh informácií, ktoré sú súčasťou skriptu.
